docs/notebooks/basic-scrna-tutorial.ipynb

```diff
@@ -0,0 +1,1193 @@
+{
```

Comment:
<p>pls link packages and possible the benchmark paper or chapter from the best practice book</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scverse-tutorials/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>